### PR TITLE
add name

### PR DESCRIPTION
**Pull request by sotarokashiuchi**

### CODE REVIEW

**Review by gpgpc0 — COMMENTED** [review]
This is a Comment

**Review by tachyon0306 — COMMENTED** [review]
test

**Review by gpgpc0 — CHANGES_REQUESTED** [review]
特に変えなくて大丈夫()

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 ## 開発者
 - 各受講生の名前を記入してください。
+- KashiuchiSotaro
```
Comment (tachyon0306) [inline README.md:9]: test

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
...
 
 ## 数値解析ソフト
 - 与えられた整数の特性を解析し、表示するソフト
```
Comment (tachyon0306) [inline README.md:12]: bennri

**Review by tachyon0306 — CHANGES_REQUESTED** [review]

**Review by gpgpc0 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sotarokashiuchi) [reply README.md:9]: Complete

**Review by tachyon0306 — APPROVED** [review]

**Review by gpgpc0 — APPROVED** [review]
PR is OK.

**Comment by gpgpc0** [comment]
Test

**Pull request merged**